src/Symfony/Bridge/PhpUnit/CoverageListener.php

```diff
@@ -29,7 +29,7 @@ class CoverageListener implements TestListener
     public function __construct(callable $sutFqcnResolver = null, bool $warningOnSutNotFound = false)
     {
         $this->sutFqcnResolver = $sutFqcnResolver ?? static function (Test $test): ?string {
-            $class = \get_class($test);
+            $class = $test::class;
```

Comment:
Should be reverted as PHPUnit must still be compatible with PHP 7.1 (same for all files in this bridge).